src/beacon/app/routes/analytics.py

```
@@ -0,0 +1,481 @@
+@router.get("/system-health")
+async def get_system_health(
+    *,
+    db: Session = Depends(get_db)
+):
+    """
+    Get overall system health metrics
+    """
+    # Device statistics
+    total_devices = db.exec(select(func.count(Device.device_key))).first()
+    active_devices = db.exec(
+        select(func.count(Device.device_key)).where(Device.is_active == True)
```

Ruff (0.12.2)

Avoid equality comparisons to `True`; use `Device.is_active:` for truth checks

Replace with `Device.is_active`

(E712)